### ISSUE TITLE
Banner colour pairing fails accessibility checks

### ISSUE THREAD

**Issue by HelenWDTK** [mysociety/whatdotheyknow-theme#1920]
The white on green text is hard for me to read, and fails contrast tests.

<img width="627" alt="Screenshot 2024-09-20 at 09 54 40" src="https://github.com/user-attachments/assets/4fc117c2-b5fb-42db-92ca-55970621c2a7">
